Update setup scripts with s3 read access only account

osbornjd opened the pull request:
Updates the setup script to check for S3 read access account setup:

If the endpoint has already been configured, does nothing
If it doesn't, configures the endpoint with the alias `eicS3` such that users can then access S3 via something like:

`mcs3 ls eicS3/eic/blahblah`

pinkenburg reviewed (changes requested):
The echo "" in the csh script will cause problems in rcf. We use winsshfs to mount rcf filesystems to our laptops. If anything prints anything out during the login this breaks. Since often users source the setup script in their .cshrc or .login the setup script must not echo anything - not even an end of line

osbornjd commented:
Oops, I mixed up the csh and sh code anyway in copying over. I fixed this but when the S3 endpoint is setup S3 prints something out by default - nothing I can do about this. Does this mean this can't actually go in the setup script? e.g. what I mean is
```
mcs3: Configuration written to `/phenix/u/jdosbo/.mcs3/config.json`. Please update your access credentials.
mcs3: Successfully created `/phenix/u/jdosbo/.mcs3/share`.
mcs3: Initialized share uploads `/phenix/u/jdosbo/.mcs3/share/uploads.json` file.
mcs3: Initialized share downloads `/phenix/u/jdosbo/.mcs3/share/downloads.json` file.
Added `eicS3` successfully.
```
is printed by default after `mcs3 config host ...`

pinkenburg commented:
This PR was for the gcc-8.3 branch, not the master branch
